lib/log_blackbox.c

```diff
@@ -161,19 +161,52 @@ qb_log_blackbox_open(struct qb_log_target *t)
 	return 0;
 }
 
+/* This is designed to look as much like the ringbuffer header
+ * as possible so that we can distinguish an old RB dump
+ * from a new one with this header.
+ */
+
+struct _blackbox_file_header {
```

Comment:
Because this structure is used directly in fwrite it should be marked as a `packed` (because `qb_rb_write_to_file` is saving field by field).

Yet another crazy idea (please ignore it if you feel like I had already too many of them :) ), what about storing it in network order so blackbox is "portable" between different endian systems?

Reply:
I'm not convinced it's worth the effort TBH.

Reply:
Understood (and kind of agree, we can always ask for text dump).